src/rebar_prv_gradualizer.erl

```diff
@@ -93,7 +99,9 @@ files_to_check(App) ->
             not lists:member(File, ExpandedExclude)
         end, ExpandedFiles).
 
--spec format_error(any()) -> string().
+-spec format_error(any()) -> iolist().
```

Comment:
The documentation suggests this could be a iolist(). https://www.rebar3.org/docs/tutorials/building_plugins/